stack.yaml

```diff
@@ -7,7 +7,7 @@ flags:
 extra-package-dbs: []
 
 packages:
-- '.'
+- cardano-sl
```

Comment:
I used another name because at first I had some issues that I incorrectly thought were related to it being `node`, this was a placeholder.